tests/monitoring_integration_test.py

```diff
@@ -106,6 +106,56 @@ def testUserExpDashboard(self):
 				
 		self.assertTrue(found_userexp_dashboard)
 
+class TestCustomService(unittest.TestCase):
```

Comment:
How are the tests being invoked? In other words, they should be added to the CI :)

Reply:
+1  They won't get run consistently otherwise.

Reply:
yup!! Will be adding to these to the CI as my first priority task for next sprint!